meteor/client/ui/Settings/ShowStyle/VariantSettings.tsx

```diff
@@ -79,61 +80,62 @@ export const ShowStyleVariantsSettings = withTranslation()(
 					return
 				}
 
-				this.importVariantsFromArray(newVariants)
+				this.importShowStyleVariantsFromArray(newShowStyleVariants)
 			}
 			reader.readAsText(file)
 		}
 
-		importVariantsFromArray = (variants: ShowStyleVariant[]) => {
+		private importShowStyleVariantsFromArray = (showStyleVariants: ShowStyleVariant[]): void => {
 			const { t } = this.props
-			_.forEach(variants, (entry: ShowStyleVariant) => {
-				MeteorCall.showstyles.insertShowStyleVariantWithBlueprint(entry, entry._id).catch(() => {
+			showStyleVariants.forEach((showStyleVariant: ShowStyleVariant) => {
+				MeteorCall.showstyles.insertShowStyleVariantWithProperties(showStyleVariant, showStyleVariant._id).catch(() => {
 					NotificationCenter.push(
 						new Notification(
 							undefined,
 							NoticeLevel.WARNING,
-							t('Failed to import Variant {{name}}. Make sure it is not already imported.', { name: entry.name }),
+							t('Failed to import Variant {{name}}. Make sure it is not already imported.', {
+								name: showStyleVariant.name,
+							}),
 							'VariantSettings'
 						)
 					)
 				})
 			})
 		}
 
-		copyVariant = (showStyleVariant: ShowStyleVariant) => {
-			MeteorCall.showstyles.insertShowStyleVariantWithBlueprint(showStyleVariant).catch(console.error)
+		private copyShowStyleVariant = (showStyleVariant: ShowStyleVariant): void => {
+			showStyleVariant.name = `Copy of ${showStyleVariant.name}`
+			MeteorCall.showstyles.insertShowStyleVariantWithProperties(showStyleVariant).catch(logger.warn)
 		}
 
-		downloadVariant = (showstyleVariant: ShowStyleVariant) => {
-			const variantArray = [showstyleVariant]
+		private downloadShowStyleVariant = (showStyleVariant: ShowStyleVariant): void => {
+			const variantArray = [showStyleVariant]
```

Comment:
Minor thing. You don't have to call your variables `xArray`. Using the plural form of the entity is fine. So here it would be `showStyleVariants`

Reply:
Changed it! [2b97864](https://github.com/tv2/tv-automation-server-core/pull/188/commits/2b9786455c2556827cfaf626cf77593836cbbc76)